src/main/java/com/teksystems/TekcampExerciseSpring/Controllers/UserController.java

```diff
@@ -0,0 +1,79 @@
+package com.teksystems.TekcampExerciseSpring.Controllers;
+
+import java.util.Optional;
+
+import org.springframework.beans.BeanUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.teksystems.TekcampExerciseSpring.Models.UserEntity;
+import com.teksystems.TekcampExerciseSpring.Models.Request.UserRequest;
+import com.teksystems.TekcampExerciseSpring.Models.Response.UserResponse;
+import com.teksystems.TekcampExerciseSpring.Services.UserService;
+import com.teksystems.TekcampExerciseSpring.Shared.Dto.UserDto;
+import com.teksystems.TekcampExerciseSpring.Validator.UserValidator;
+
+@RestController
+@RequestMapping(path = "users")
+public class UserController {
+	
+	private final UserService userServ;
+	
+	public UserController(UserService userServ) {
+		this.userServ = userServ;
+	}
+	
+	@GetMapping()
+	public Iterable<UserEntity> displayUsers(
```

Comment:
convention here is to name this method getUsers, not display, because you don't know for sure what your user is going to do with the information on the front end. They may never display the information at all. (nitpick I know)